src/logger.rs

```diff
@@ -1,33 +1,31 @@
-use fern;
 pub use log::LevelFilter;
-use std::env;
-use std::io;
+
+use fern;
+use std::{env, io, path::PathBuf, str::FromStr};
 
 /// Logger configuration object.
-#[derive(Clone, Copy)]
+#[derive(Clone)]
```

Comment:
Ah, another idea, this should derive `Serialize` so we change logging without recompiling.

Reply:
Do you mean both `Serialize` and `Deserialize`? Also, `serde` is currently listed as a dev dep, should I promote it then?

Reply:
Be careful with this as the current ways of deserializing in Amethyst rely on the log macros for error handling, meaning that you wouldn't get an error message when initializing the logger if your config is wrong.

Reply:
Actually, `LevelFilter` doesn't implement `Serialize`, so I'm not sure if this is possible (I should probably note that I haven't worked with `serde` before)

Reply:
@magnonellie That's only in `Config`, which shouldn't be used here.